### DISCUSSION TITLE
How to disable anything text related in completions?

### DISCUSSION THREAD

**Discussion by voodookiidoo** [saghen/blink.cmp#586] [Q&A]
I really want to disable completions for the text tokens, i've looked at config but still don't understand how to turn it off. I guess the buffers must be disabled somehow, but yet i don't know how to do it
<img width="748" alt="image" src="https://github.com/user-attachments/assets/ffadd245-bf5e-4223-bf05-5cdf5ebbbc0c" />
Like those 'abc' suggestions

**Comment by saghen** [comment]
https://github.com/Saghen/blink.cmp/issues/484#issuecomment-2530068626
